account_invoice_alternate_payer/models/account_move.py

```diff
@@ -71,84 +62,89 @@ def _sync_dynamic_line(
             yield
 
     def _compute_payments_widget_to_reconcile_info(self):
-        super_moves = self.filtered(lambda r: not r.alternate_payer_id)
-        for move in self - super_moves:
+        super()._compute_payments_widget_to_reconcile_info()
```

Comment:
@xaviedoanhduy only call super for records which has no value of alternate_payer_id